src/main/java/model/RandomPhrase.java

```diff
@@ -0,0 +1,28 @@
+package model;
+
+import java.util.ArrayList;
+
+public class RandomPhrase {
+    private boolean needInit = true;
+
+    public String getPhrase(){
+        ArrayList<String> phrases = new ArrayList<>();
```

Comment:
Please, use abstraction instead implementation where it's possible
```suggestion
        List<String> phrases = new ArrayList<>();
```

Reply:
This is the common advice, thank you. But I have forgotten, why do we need to declare this way. I googled it, but i didn't find the complete and proper explanation. There are only advices to do this way. Can you explain me, why it is better to use interface instead class? 

My only thought is easier return: when we declare List<String> we can return everything we want - ArrayList<String> or LinkedList<String>. But is it so important? 

I know, that we need to declare List, it is the law, but I have forgotten this law because I have no idea why it is the best way.